core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/rest/urlparsers/V4ExistingResourceByCrnOrNameRestUrlParser.java

```diff
@@ -18,11 +18,11 @@
     // v4/{workspaceId}/blueprints/name/{name}
     // v4/{workspaceId}/blueprints/crn/{name}
     private static final Pattern PATTERN = Pattern
-            .compile("v4/(\\d+)/(blueprints|image_catalogs|recipes|stacks|distrox|cluster_templates)/(?:name|crn)/([^/]+)");
+            .compile("v4/(\\d+)/(blueprints|image_catalogs|recipes|stacks|distrox|cluster_templates)/(?:name|crn)/([^/]+)(/\\w+)?");
```

Comment:
could you add some test for this change? either via `RestUrlParserTest` or you could create a new test class
thanks

Reply:
The RestUrlParserTest already catches this as part of the test.
testIfRestUrlParserMissing goes through every URL pattern defined in the controllers and makes sure all of the defined URLs have a parser.  So for every existing stack URla don't the new one get caught and the pattern applied to.  The reason for the change to V4ExistingResourceByCrnOrNameRestUrlParser.java was that the new URL pattern with crn and an action on the crn was failing.  So testIfRestUrlParserMissing checks all URL patterns against the V4ExistingResourceByCrnOrNameRestUrlParser Pattern.

Reply:
thanks for clarification!